spec/zoho_hub/base_record_spec.rb

```diff
@@ -55,6 +41,48 @@
     end
   end
 
+  describe '#build_response' do
+    context 'with an empty string and a "false" boolean' do
+      let(:body) { { data: [{ My_String: '', My_Bool: false }] } }
+
+      it 'correctly construct the record' do
+        response = test_class.build_response(body)
+        record = test_class.new(response.data.first)
+
+        expect(record.my_string).to eq('')
+        expect(record.my_bool).to eq(false)
+      end
+    end
+  end
+
+  describe '#blueprint_transition' do
+    let(:test_instance) do
+      described_class.new
+    end
+
+    before do
+      allow(described_class).to receive(:to_s).and_return('Leads')
+      allow(test_instance).to receive(:id).and_return('123456789')
```

Comment:
Can't you set that during the `test_instance` definition? (`described_class.new(id: '123456789')`)